File: General/largest_perimeter_of_triangle.py
class Solution:
    def largestPerimeter(self, nums: List[int]) -> int:
        arr = sorted(nums)
        for i in range(len(nums)-3,-1,-1):
            print(arr[i],arr[i+1],arr[i+2])
            if(arr[i+1]+arr[i]>arr[i+2]):
                return arr[i+1]+arr[i+2]+arr[i]
        return 0

File: General/largest_perimeter_of_triangle.cpp
class Solution {
public:
    int largestPerimeter(vector<int>& nums) {
        int n = sizeof(nums)/sizeof(nums);
        sort(nums.begin(),nums.end());
        for (int i=nums.size()-3; i>-1; i--){
            if(nums[i+1]+nums[i] > nums[i+2] ){
                return nums[i+1]+nums[i+2]+nums[i];
            }
        }
        return 0;
    }
};